packages/block-library/src/video/edit.js

```diff
@@ -200,6 +201,11 @@ class VideoEdit extends Component {
 							onChange={ this.toggleAttribute( 'controls' ) }
 							checked={ controls }
 						/>
+						<ToggleControl
+							label={ __( 'Playsinline' ) }
```

Comment:
```suggestion
							label={ __( 'Play inline' ) }
```

Reply:
Thank you so much @swissspidy ❤️ I was quite confused about what to set the label as 😓 Fixed in this [commit](https://github.com/WordPress/gutenberg/pull/14500/commits/c855b7f6cb8dda022809b3dc74a5c8fbb8de5d93).